doc/user_guide/transformations.rst

```diff
@@ -316,6 +316,16 @@ can be found in the API-specific sections).
 
 ####
 
+.. autoclass:: psyclone.psyir.transformations.DebugChecksumTrans
+      :members: apply
+      :no-index:
+
+.. warning::  This transformation generates code that will only work with .F90
```

Comment:
You don't need this one as the documentation generated from the docstring by `.. autoclass` includes the one from there.

Reply:
Ah ok, I'll remove it.